internal/api/impl/auth/auth.go

```diff
@@ -71,12 +71,13 @@ func getRootURL(r *http.Request) url.URL {
 			u.Scheme = "https"
 		}
 	}
+	u.Path = apiPrefix
 	return u
 }
 
-func getRedirectURI(r *http.Request, authKind string, slugID string) string {
-	rd := getRootURL(r)
-	rd.Path = fmt.Sprintf("%s/%s/%s/%s/%s", utils.APIPrefix, utils.PathAuthProviders, authKind, slugID, utils.PathCallback)
+func getRedirectURI(r *http.Request, authKind, slugID, apiPrefix string) string {
+	rd := getRootURL(r, apiPrefix)
+	rd.Path += fmt.Sprintf("%s/%s/%s/%s/%s", utils.APIPrefix, utils.PathAuthProviders, authKind, slugID, utils.PathCallback)
```

Comment:
in case `getRootURL` is used only in `getRedirectURI` it will be simpler if the apiPrefix is appended here

Reply:
no getRootURL is used in the logout. I need the api prefix there